### PR TITLE
Modified test may 18

### PR DESCRIPTION
**Pull request by manjunath-grl**
Summary:
Scripts added
TC-ULABEL-2.4
TC-ULABEL-2.5
Modified scripts
TC-CC-9.1
TC-CC-9.2
TC-CC-9.3
TC-FLW-2.1
TC-GC-1.1
TC-GC-2.1
TC-ILL-2.1
TC-LVL-2.2
TC-MC-1.5
TC-MC-1.7
TC-MC-1.10
TC-OO-2.3
TC-PSCFG-1.1
TC-RH-1.1
TC-TM-11
TC-TSTAT-1.1
TC-TSUIC-1.1
TC-TSUIC-2.1
added auto generated files

Attached execution logs:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/8806783/Execution_log.txt)

### CODE REVIEW

File: examples/darwin-framework-tool/templates/tests/tests.js [hunk 1]
```diff
@@ -57,6 +57,15 @@ function getTests() {
   // TODO: Test_TC_DIAG_TH_NW_2_3 does not work on Darwin for now.
   tests.disable('Test_TC_DIAG_TH_NW_2_3');
 
+  // TODO: Test_TC_CC_9_1 does not work on Darwin for now.
+  tests.disable('Test_TC_CC_9_1');
```
Comment (krypton36) [inline examples/darwin-framework-tool/templates/tests/tests.js:61]: @manjunath-grl can you please file an issue?

**Review by krypton36 — APPROVED** [review]

File: examples/darwin-framework-tool/templates/tests/tests.js (hunk 1 above)
Comment (manjunath-grl) [reply examples/darwin-framework-tool/templates/tests/tests.js:61]: @krypton36 sure

**Review by woody-apple — APPROVED** [review]
Fast tracking test updates

**Comment by woody-apple** [comment]
Tagging @raju-apple to take a peek at conflicts

**Comment by raju-apple** [comment]
> Tagging @raju-apple to take a peek at conflicts

@woody-apple @bzbarsky-apple All green now , can one of you merge it in please before we get more conflicts :) ?

**Pull request merged**